src/test/java/org/fundacionjala/sfdc/stepDefinition/CreateContactSteps.java

```diff
@@ -0,0 +1,72 @@
+package org.fundacionjala.sfdc.stepDefinition;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+/**
+ * CreateContactSteps.java.
+ * Class for steps createContact.feature.
+ */
+public class CreateContactSteps {
+
+
+    /**
+     * Given step.
+     */
+    @Given("^I am going to app launcher page$")
+    public void iAmGoingToAppLauncherPage() {
+        System.out.println("I go to app launcher button.");
+
+
+    }
+
+    /**
+     * Given step.
+     */
+    @Given("^I am on the new contact page$")
+    public void iAmOnTheNewContactPage() {
+        System.out.println("This Step open the Firefox and go to login page of the application.");
+
+       /* contact = new CreateContacts(web);
+        contact.creation("naranja", "mandarina");*/
+
+    }
+
+    /**
+     * When step.
+     */
+    @When("^I fill in First name with Ariel$")
+    public void iFillInFirstNameWithAriel() {
+        System.out.println("This step enter the Username and Password on the login page.");
+
+    }
+
+    /**
+     * And step.
+     */
+    @And("^I fill in Last Name with Gonzales$")
+    public void iFillInLastNameWithGonzales() {
+        System.out.println("Click login.");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Click login." 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)